docs/SetupGuide_Dotnet.md

```diff
@@ -21,6 +21,8 @@
       - [ICollector\<T\>/IAsyncCollector\<T\>](#icollectortiasynccollectort)
       - [Array](#array)
       - [Single Row](#single-row)
+  - [CSharp Scripting](#csharp-scripting)
```

Comment:
Also add links to the specific samples under each of the samples sections in this file